src/Templates/src/templates/maui-mobile/MauiApp.1.csproj

```diff
@@ -62,4 +62,8 @@
 		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="MS_EXT_LOG_DEBUG_VERSION" />
 	</ItemGroup>
 
+	<!-- Apple Privacy Manifest file, more information: https://aka.ms/maui-privacy-manifest -->
+	<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">
+		<BundleResource Include="Platforms\iOS\PrivacyInfo.xcprivacy" LogicalName="PrivacyInfo.xcprivacy" />
```

Comment:
I get this when using `LogicalName` in my project:
![image](https://github.com/dotnet/maui/assets/1378165/5009e475-4e6d-48df-a205-dc834ac2cef4)